ui/main/src/app/components/navbar/navbar.component.ts

```diff
@@ -51,29 +52,32 @@ export class NavbarComponent implements OnInit {
                 _.fill(this.expandedMenu,false);
             }));
         this.store.dispatch(new LoadMenu());
+
         this.store.select(buildConfigSelector('logo.base64')).subscribe(
             data => {
               if (data) this.customLogo = `data:image/svg+xml;base64,${data}`
             }
         );
+
         this.store.select(buildConfigSelector('logo.height')).subscribe(
             height => {
               if (height) this.height = height;
             }
         );
+
         this.store.select(buildConfigSelector('logo.width')).subscribe(
             width => {
               if (width) this.width = width;
             }
         );
         this.store.select(buildConfigSelector('logo.limitSize')).subscribe(
-            limitSize => {
-              if (limitSize) this.limitSize = limitSize;
+            (limitSize:boolean) => {
+                // BE CAREFUL, as a boolean it has to be test with undefined value to know if it has been setted.
```

Comment:
`<grammar rigorist>`

*  "it has to be _tested_ ";
* _set_ instead of _setted_.

`</grammar rigorist>`

Reply:
fixed